framework/resource/retryresource/basic_resource.go

```diff
@@ -0,0 +1,116 @@
+package retryresource
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/cenkalti/backoff"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"github.com/giantswarm/operatorkit/framework"
+)
+
+type basicResourceConfig struct {
+	Logger micrologger.Logger
+	//Resource framework.Resource
+	Resource interface {
```

Comment:
Why is this now called `Resource` and not `Ops`? This should actually be consistent with the other resources IMO. Also, why don't we call this `BasicResource` then because the file is called like this, which makes sense to me.

Reply:
If we should call anything `Ops` it should be that: https://github.com/giantswarm/operatorkit/pull/153/files#diff-9bd9495915276bffdd4cd4924c8b17e3R29

Type here is called `basicResource` after the file: https://github.com/giantswarm/operatorkit/pull/153/files#diff-919a9c5983f8f6d9a46da6822dcf7f85R28

Reply:
Yeah I still do not get the Ops naming and why this is now different here.